.github/actions/setup-deps/action.yaml

```diff
@@ -82,6 +82,8 @@ inputs:
     default: 'seaborn>=0.7.0'
   tidynamics:
     default: 'tidynamics>=1.0.0'
+  imdclient:
+    default: 'imdclient'
```

Comment:
It's going to take time to get imdclient to a stage where the imdclient package does not actually affect MDAnalysis. 

Is there a way that we could temporarily (for initial CI testing) install imdclient from a branch or tarball, e.g., in a `pip` section? Then we could fairly rapidly create a preliminary (unpublished) imdclient package without IMDReader.

Reply:
By initial CI testing, do you mean "in this PR"?

There's a pip section just below, which should work if you put in the git location for pip install, but also you can just temporarily modify the CI script to do an additional pip install if it's for "testing within the PR itself".

If it's "after merge", this would require more discussion.

Reply:
Just for _right now_ to bootstrap the PR.

I don't want to merge without a solid conda-forge imdclient package in place.